d1lod/d1lod/client.py

```diff
@@ -18,8 +21,11 @@
 
 
 FORMAT_MAP = {
+    "eml://ecoinformatics.org/eml-2.0.0": EMLProcessor,
+    "eml://ecoinformatics.org/eml-2.0.1": EML201Processor,
+    "eml://ecoinformatics.org/eml-2.1.0": EML210Processor,
     "eml://ecoinformatics.org/eml-2.1.1": EML211Processor,
-    "https://eml.ecoinformatics.org/eml-2.2.0": EML220Processor,
+    "eml://ecoinformatics.org/eml-2.2.0": EML220Processor,
```

Comment:
This should be `https://eml.ecoinformatics.org/eml-2.2.0`.

Reply:
Thanks for catching it; fixed in [40e8016](https://github.com/DataONEorg/slinky/pull/69/commits/40e80169644898bf9ab2c92e7b4fed8eadb6b14e)